R/scripts.R

```diff
@@ -88,8 +88,15 @@ run_civis <- function(expr, ...) {
 #' Run a template script
 #' @param id id of the template script.
 #' @param arguments list of arguments to the script.
+#' @param JSONValue bool (default FALSE) If true, returns the 
+#'                  JSON values instead of the file_ids
 #' @param ... additional arguments to \code{scripts_post_custom}
-#' @return File ids of any run outputs are returned.
+#' @return If JSONValue is FALSE, File ids of any run outputs are returned.
+#'         If JSONValue is TRUE, JSON values of first JSON run output is returned.
+#'           If there are no JSON outputs, nothing is returned
+#'           If there are more than 1 JSON outputs, error message is printed 
+#'             and nothing is returned.
```

Comment:
This is stale since your update to the behavior for > 1 JSON output

Reply:
and.. i think we're there now.